garrysmod/lua/includes/extensions/util.lua

```diff
@@ -206,6 +209,15 @@ local T =
 
 		return self.endtime == nil or self.endtime <= CurTime()
 
+	end,
+
+	--
+	-- Returns the amount of time that has passed since the Timer was started
+	--
+	GetElaspedTime = function( self )
+
+		return self:Started() and CurTime() - self.m_timestamp or 99999.9

```

Comment:
I am not sure it makes sense to return 99999.9 for a stopped timer. Perhaps `-1` will be a more logical return being an "invalid time" rather than returning a large value that may or may not be the real elapsed time.

Reply:
Come to think of it, shouldn't it just return the elapsed time at the moment the timer was stopped anyway?

Reply:
I'm basing this PR on the IntervalTimer object in the Source 2013 SDK. The GetElaspedTime in there internally returns 99999.9. I don't know why Valve did that, but I did change it to -1.0 as you suggested. When the timer is reset, Valve sets m_timestamp to -1.0. If you want to make any other changes, you should have access to edit the PR.